src/components/screen/Setting/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { Alert, SectionList, SectionListData } from 'react-native';
+import {
+  Container,
+  HeaderContainer,
+  ItemContainer,
+  ItemIcon,
+  ItemLabel,
+  SectionHeader,
+} from './styles';
+import { DefaultNavigationProps, SettingsOption } from '../../../types';
+import {
+  DefaultTheme, withTheme,
+} from 'styled-components/native';
+import React, { useRef, useState } from 'react';
+import ChangePwView from './ChangePwView';
+import { FontAwesome } from '@expo/vector-icons';
+import { IC_FACEBOOK } from '../../../utils/Icons';
+import Modal from 'react-native-modalbox';
+import { getString } from '../../../../STRINGS';
+
+function SectionItem(option: SettingsOption, theme: DefaultTheme): React.ReactElement {
+  return (
+    <ItemContainer onPress={option.onPress} testID={option.testID}>
+      {option.icon ? <ItemIcon source={option.icon} /> : null}
+      <ItemLabel>{option.label}</ItemLabel>
+      <FontAwesome name="angle-right" size={32} color={theme.fontColor} />
+    </ItemContainer>
+  );
+}
+
+interface Props {
+  navigation: DefaultNavigationProps<'Setting'>;
```

Comment:
Good one here. You are using it correctly.

Reply:
Thanks. But, is it needed? In this https://github.com/dooboolab/hackatalk-mobile/pull/58#discussion_r347398884 you said that it'd be better to use `const navigation = useNavigation();`

Reply:
This is needed if you want to strictly check the state param passed down to the `Setting` screen.

Oh yes, and sorry for confusing you. I wanted to tell you that `useNavigation` would be efficient in such cases if you need the navigation props in `shared` components. Then you won't have to pass down the `navigation` props every time. In this case, `useNavigation` is not a good idea.